src/nft/components/collection/UnavailableCollectionPage.tsx

```diff
@@ -0,0 +1,67 @@
+import { Trans } from '@lingui/macro'
+import Column from 'components/Column'
+import { darken } from 'polished'
+import { AlertTriangle } from 'react-feather'
+import { Link } from 'react-router-dom'
+import styled, { css, useTheme } from 'styled-components'
+import { ThemedText } from 'theme/components'
+
+const Container = styled(Column)`
+  height: 75vh;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: 48px;
+  gap: 8px;
+`
+const LinkStyles = css`
+  color: ${({ theme }) => theme.accent1};
+  text-decoration: none;
+  &:hover,
+  &:focus {
+    color: ${({ theme }) => darken(0.1, theme.accent1)};
+  }
+`
+const InternalLink = styled(Link)`
+  ${LinkStyles};
+`
+const ExternalLink = styled.a`
+  ${LinkStyles};
+`
+export function UnavailableCollectionPage({ isBlocked }: { isBlocked?: boolean }) {
+  const theme = useTheme()
+  return isBlocked ? (
+    <Container>
+      <AlertTriangle
+        width="48px"
+        height="48px"
+        stroke={theme.background}
+        strokeWidth="1px"
+        fill={theme.critical}
+        data-testid="alert-icon"
+      />
+      <ThemedText.HeadlineMedium>
+        <Trans>This collection is blocked</Trans>
+      </ThemedText.HeadlineMedium>
+      <ExternalLink
+        href="https://support.uniswap.org/hc/en-us/articles/18783694078989-Unsupported-Token-Policy"
+        target="_blank"
```

Comment:
Wasn't sure about this, so did a little research. Looks like the default value for `target` is `_self`, so we should set this to `_blank`. It may be possible to remove `noopener`, since many major browsers now automatically use the behavior of `rel="noopener"` for any `target="_blank"` link. However, it may not be all browsers. Since `noopener` can prevent a critical security vulnerability, I think it's better to include it.

I like your idea of simplifying this though. I'm going to add a reusable `ExternalLink` component. I also agree that we should move the URLs to a constants file.

Reply:
oh you're using a `a` tag here, i thought you were importing `ExternalLink` component.

https://github.com/Uniswap/interface/blob/main/src/theme/components/index.tsx#L165